.github/workflows/build-nym-vpn-app-macos.yml

```
@@ -0,0 +1,781 @@
+        uses: actions/upload-artifact@v7
+        with:
+          name: ${{ env.UPLOAD_DIR_MACOS }}
+          path: ${{ env.UPLOAD_DIR_MACOS }}
+          if-no-files-found: error
+          retention-days: 1
+
+      - name: Upload macOS dSYMs to Sentry (nightly)
+        if: ${{ env.RELEASE_TYPE == 'qa' }}
+        working-directory: nym-vpn-apple
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.APPLE_SENTRY_CLI_AUTH_TOKEN }}
```

actionlint (1.7.12)

[error] property "apple_sentry_cli_auth_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; app_store_connect_api_issuer_id: string; app_store_connect_api_key: string; app_store_connect_api_private_key: string; apple_developer_id_application_cert: string; apple_developer_id_application_cert_pass: string; apple_developer_id_installer_cert: string; apple_developer_id_installer_cert_pass: string; apple_team_id: string; crowdin_personal_token: string; crowdin_project_id: string; github_token: string}

(expression)